README_installation_in_vagrant.md

```diff
@@ -0,0 +1,101 @@
+Open Disclosure backend installation in Vagrant in 6 steps
+=================
+
+Scripts to build Open Disclosure's back end in a Vagrant virtual machine.
+This has been tested on a Mac running Yosemite. From what I understand, Virtual box and Vagrant work in Windows as well, and most of the commands are the same, though, I have not tested it in Windows.
+
+To install the backend:
+
+####1) Install Virtual box and Vagrant.
+
+See [https://www.virtualbox.org/](https://www.virtualbox.org/) for virtual box.
+	
+See [http://www.vagrantup.com/](http://www.vagrantup.com/) for Vagrant.
+		
+-----------------------
+###Three important notes on Vagrant usage:
+	
+######a) Never run Vagrant commands from your home machine as root :)
+  ie, if you mistakenly run anything with the pattern 
+	
+		host machine prompt> sudo vagrant xxx
+		
+  you will probably need to read [this](http://stackoverflow.com/questions/25652769/should-vagrant-require-sudo-for-each-command) to make everything   work right again.
+	
+  Running code as root inside of the Vagrant vitual machine, ( ie: after >vagrant ssh and before ctrl-d ), is fine. 
+	
+######b) Starting the Vagrant virtual machine after it is shut down 
+  To restart a virtual box, you must first go to the directory where you first initilized and installed the Vagrant box, then use the 'vagrant up' command. If you type this command:
+		
+		host machine prompt> vagrant up
+		
+  in the wrong directory it will cause much confusion and problems. 
+	
+######c) Re-installing the virtual box
+  If you are re-installing the back end and wish to re-install the vagrant instance, you need to first destroy the previously installed Vagrant box using these commands, run from the directory where the original Vagrant box was installed. 
+  
+	> vagrant destroy
+	
+  Then remove the Vagrant config file found in the installation directory (it is named "Vagrantfile", no extension). 
+  
+	> rm Vagrantfile
+
+-----------------------
+####2) Setup the Vagrant folder
+
+######a) Copy this git repository to a folder on your computer.
+	
+######b) Navigate to the git repository's folder; this will be used as the Vagrant install folder.
+  
+####3) Establish the Virtual box and install the server
+At a terminal command prompt run these commands:
+  
+	host machine prompt> vagrant box add ubuntu14 https://cloud-images.ubuntu.com/vagrant/trusty/current/trusty-server-cloudimg-i386-vagrant-disk1.box
+	host machine prompt> vagrant init ubuntu14
```

Comment:
Ah, the problem with the filename is that you shouldn't have this command (or the `vagrant box add` one) as part of the instructions -- they are only intended for the author of a commit that adds a `Vagrantfile` to a repository. Once committed, all it takes is `vagrant up` to provision the box. You'll notice that the `Vagrantfile` contains the box URL so that people running `vagrant up` will get the right image.